site/src/components/Conditionals/ChooseOne.stories.tsx

```diff
@@ -1,46 +1,71 @@
-import { Story } from "@storybook/react";
+import { Meta, StoryObj } from "@storybook/react";
 import { ChooseOne, Cond } from "./ChooseOne";
 
-export default {
+const meta: Meta<typeof ChooseOne> = {
   title: "components/Conditionals/ChooseOne",
   component: ChooseOne,
-  subcomponents: { Cond },
 };
 
-export const FirstIsTrue: Story = () => (
-  <ChooseOne>
-    <Cond condition>The first one shows.</Cond>
-    <Cond condition={false}>The second one does not show.</Cond>
-    <Cond>The default does not show.</Cond>
-  </ChooseOne>
-);
+export default meta;
+type Story = StoryObj<typeof ChooseOne>;
 
-export const SecondIsTrue: Story = () => (
-  <ChooseOne>
-    <Cond condition={false}>The first one does not show.</Cond>
-    <Cond condition>The second one shows.</Cond>
-    <Cond>The default does not show.</Cond>
-  </ChooseOne>
-);
+export const FirstIsTrue: Story = {
+  args: {
+    children: [
+      <Cond key="1" condition>
+        The first one shows.
+      </Cond>,
+      <Cond key="2" condition={false}>
+        The second one does not show.
+      </Cond>,
+      <Cond key="3">The default does not show.</Cond>,
+    ],
```

Comment:
I liked the earlier commit where these were fragments: React will perform better with those. Does it break the `ChooseOne` component or something?

Reply:
Nops, it is just for the storybook. I didnt find a similar API to keep it as a fragment in the new Storybook v7 API